### PR TITLE
Test to check crush map before and after upgrade

### PR DESCRIPTION
**Pull request by fbalak**
Signed-off-by: Filip Balak <fbalak@redhat.com>

### CODE REVIEW

File: tests/ecosystem/upgrade/test_configuration.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    Loads CRUSH map before upgrade by `test_load_crush_map` test case.
+
+    Returns:
+        str: String consisting of CRUSH map before upgrade
+    """
+    crash_map = get_crush_map()
+    log.info(f"Pre upgrade CRASH map: {crash_map}")
+    return crash_map
+
+
```
Comment (ebenahar) [inline tests/ecosystem/upgrade/test_configuration.py]: Polarion ID?

Comment (hnallurv) [reply tests/ecosystem/upgrade/test_configuration.py]: Please change CRASH to CRUSH wherever applicable

Comment (fbalak) [reply tests/ecosystem/upgrade/test_configuration.py]: done

**Comment by petr-balogh** [comment]
Is that verified @fbalak ? If so please put verifed label

**Comment by ebenahar** [comment]
@petr-balogh can we merge this or you think we need to verify something before?

**Pull request merged**